VirusTotal.NET/VirusTotal.cs

```diff
@@ -76,6 +79,12 @@ public bool UseTLS
             }
         }
 
+        public bool FollowRedirects
+        {
+            get { return _client.FollowRedirects; }
+            set { _client.FollowRedirects = value; }
+        }
```

Comment:
This is required to be set to true as well for large uploads. Perhaps we can make that automatic so it doesn't fail.